src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -237,10 +237,15 @@ def _osd_rm(self, svc_id):
         return HandleCommandResult(stdout=completion.result_str())
 
     @_write_cli('orchestrator mds add',
-                "name=svc_arg,type=CephString",
+                "name=svc_arg,type=CephString "
```

Comment:
```suggestion
                "name=fs_name,type=CephString "
```
Let's remove svc_arg, as it doesn't give any hint about what it is supposed to be.